[Profiling] Clarify index cleanup instructions (#3179)
elastic/observability-docs: docs/en/observability/profiling-upgrade.asciidoc

You can delete existing profiling data in Kibana:

> . If you are upgrading from 8.9.0 or higher, from the navigation menu, go to *Dev Tools*, paste the following snippet and execute it. If you are upgrading from an earlier version you can skip this step.

Reviewer: ```suggestion
. If you're upgrading from 8.9.0 or later, go to *Dev Tools* from the navigation menu, and execute the following snippet. If you're upgrading from an earlier version, skip this step.

```
